docs/podman-rmi.1.md

```diff
@@ -15,10 +15,11 @@ Removes one or more locally stored images.
 
 **-all**, **-a**
 
-Remove all of the locally storage images
-**--force, -f**
+Remove all of the local storage images.
```

Comment:
I still don't like this line.

How about
Remove all images in the local storage.